dot/rpc/subscription/websocket.go

```diff
@@ -52,28 +52,27 @@ type WSConn struct {
 }
 
 // readWebsocketMessage will read and parse the message data to a string->interface{} data
-func (c *WSConn) readWebsocketMessage() ([]byte, *websocketMessage, error) {
-	_, mbytes, err := c.Wsconn.ReadMessage()
+func (c *WSConn) readWebsocketMessage() (bytes []byte, msg websocketMessage, err error) {
```

Comment:
should we keep the raw bytes inside the `websocketMessage` struct? 🤔

Reply:
I don't think so, the raw bytes don't belong to the `websocketMessage` struct as it is used to create one IMO